language/test/Language_test.rb

```diff
@@ -459,4 +455,159 @@
       expect(subject.main_language("csb_PL")).to eq "csb"
     end
   end
+
+  describe "#Export" do
+    it "returns map with language" do
+      subject.language = "cs_CZ.utf8"
+      expect(subject.Export).to include("language" => "cs_CZ.utf8")
+    end
+
+    it "returns map with installed languages" do
+      subject.languages = "cs_CZ,en_US"
+      expect(subject.Export).to include("languages" => "cs_CZ,en_US")
+    end
+
+    it "returns map with use_uth8 if utf is not used" do
+      subject.SetExpertValues("use_utf8" => false)
+      expect(subject.Export).to include("use_utf8" => false)
+    end
+  end
+
+  describe "#Import" do
+    it "sets language from map" do
+      subject.Import("language" => "de_DE")
+
+      expect(subject.language).to eq "de_DE"
+    end
+
+    it "sets utf-8 encoding from map" do
+      subject.Import("use_utf8" => false)
+
+      expect(subject.GetExpertValues["use_utf8"]).to eq false
+    end
+
+    it "sets installed languages from map" do
+      subject.Import("languages" => "de_DE,cs_CZ", "language" => "de_DE")
+
+      expect(subject.languages).to eq "de_DE,cs_CZ"
+    end
+
+    it "adds to installed languages language from map" do
+      subject.Import("languages" => "cs_CZ", "language" => "de_DE")
+
+      expect(subject.languages).to eq "cs_CZ,de_DE"
+    end
+
+    # TODO: Autoinst specific behavior
```

Comment:
np: What about adding an empty "it" so we get the "pending" item in the results?